libraries/AP_Landing/AP_Landing.cpp

```diff
@@ -46,13 +46,16 @@ const AP_Param::GroupInfo AP_Landing::var_info[] = {
     // @DisplayName: Landing Pitch
     // @Description: Used in autoland to give the minimum pitch in the final stage of landing (after the flare). This parameter can be used to ensure that the final landing attitude is appropriate for the type of undercarriage on the aircraft. Note that it is a minimum pitch only - the landing code will control pitch above this value to try to achieve the configured landing sink rate.
     // @Units: cdeg
+    // @Range: -4500 4500
```

Comment:
The range is really excessive here, the net effect is it's hard to make a useful control for the user, the range should be reduced.

Reply:
fixed